File: C#/Estrutura repetitiva/soma.cs
// Variável
int troca, soma, i, num1, num2;

soma = 0;

// Entrada de dados
Console.Write("Digite um número: ");
num1 = int.Parse(Console.ReadLine());

Console.Write("Digite outro número: ");
num2 = int.Parse(Console.ReadLine());

// Estrutura condicional
if (num1 > num2)
{
    troca = num1;
    num1 = num2;
    num2 = troca;
}

// Estrutura repetitiva
for (i = num1+1; i < num2; i++)
{
    // Estrutura condicional
    if (i % 2 != 0)
    {
        soma += i;
    }
}

// Saída de dados
Console.Write("A soma dos impares de " + num1 + " e " + num2 + " é " + soma);

File: Python/Arranjos vetoriais/soma.py
# Variável
soma = 0
media = 0

# Entrada de dados
n = int(input("Quantos números você vai digitar? "))

# Vetor
vet: [float] = [0 for x in range(n)]

# Estrutura repetitiva
for i in range(0, n):
    vet[i] = float(input("Digite um número: "))

# Saída de dados
print("\nValores:", end=" ") # Usando end evita que ele vá para linha debaixo, é automático no Python.

# Estrutura repetitiva
for i in range(0, n):
    print(f"{vet[i]:.1f}", end="  ")

# Saída de dados
print(f"\nSoma: ", end="")
# Estrutura repetitiva
for i in range(0, n):
    soma += vet[i]
# Saída de dados
print(f"{soma:.2f}")

# Saída de dados
media = soma / n
print(f"Média: {media:.2f}")